.github/workflows/linux_build_test.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           5.4.1,
           5.5.1,
         ]
+        ddl_deps : [
```

Comment:
What is the use of it?

Reply:
This is like Enable Download MOAB or Double Down?

Reply:
this only enables MOAB Download, otherwise everything work as it used to.

Reply:
this seems to [pass](https://github.com/svalinn/DAGMC/actions/runs/12199530439/job/34033629285), I'll revert the change